tests/services/orders/order_store_service/test_generic_order_store_service.py

```diff
@@ -194,106 +194,3 @@ def test_store_cancer_samples(
     assert new_link.sample.application_version.application.tag == "WGSPCFC030"
     assert new_link.sample.comment == "other Elution buffer"
     assert new_link.sample.is_tumour
-
-
-def test_store_existing_single_sample_from_trio(
```

Comment:
Both removed tests in this file started failing because they called the same storing method twice, resulting in a uniqueness constraint being broken in the Order table. Given that these methods should not be called more than once with the same ticket id I saw them fit to be removed. They can be reworked but it would be a bit annoying